beacon/beacon.go

```diff
@@ -188,8 +193,8 @@ func FromEvent(event *types.Event) Beacon {
 		Pid:            values.Get("pid"),
 		N:              values.Get("n"),
 		UserAgent:      userAgent,
-		H_CF_IPCountry: cleanupHeaderValue(header.Get("CF-IPCountry")),
-		H_CF_IPCity:    cleanupHeaderValue(header.Get("CF-IPCity")),
+		H_CF_IPCountry: country,
```

Comment:
Check my comment above.

We probably don't need H_CF_IPCountry and H_CF_IPCity om the object. Do you think that we could do refactoring and to have the calculation happening in `ConvertToRumEvent`?

I suppose that will require some refactoring and and extending the parameters that `ConvertToRumEvent` accepts.

Reply:
@ivan-stankov-golang could you check this comment as well? 

The proposal was to completely remove H_CF_IPCountry and H_CF_IPCity from the Beacon struct. Please let me know if you think that such a refactoring will make the code more clean.

Also I in the latest change set I don't see the function `cleanupHeaderValue` used anywhere anymore. We probably will need it. Let's add a test where we have headers H_CF_IPCountry and H_CF_IPCity with trailing spaces so we will keep this as a requirement for our system.